scripts/update-chart.js

```diff
@@ -83,37 +29,68 @@ const options = {
   },
   height: '100%',
   width: '100%',
-};
+});
+
+var myHeaders = new Headers();
+myHeaders.append("Content-Type", "application/json");
+
+const DEFAULT_URL = 'https://91o7sqo7s3.execute-api.us-east-1.amazonaws.com/prod/';
 
+const getYieldsFor = ({
+  year,
+  month,
+  risk,
+  portfolioType,
+  typeUniverse
+}) => {
+  var routeParam = 'recommendationYields';
+  if (portfolioType === 'harryIpsa') return chileanSmartFundYields;
+  if (portfolioType === 'wallet') routeParam = 'walletYields';
+  const url = `${DEFAULT_URL}${routeParam}`;
+  const requestOptions = {
+    method: 'POST',
+    headers: myHeaders,
+    body: JSON.stringify({ year, month, risk, typeUniverse}),
+    redirect: 'follow'
+  };
+  return fetch(url, requestOptions)
+    .then(response => response.json())
+};
 const numberToPercent = (number) => ((number) * 100).toFixed(2) + '%';
 
-const mapExtractingPercentagesWithTooltipInfo = (accumulatedYields) => accumulatedYields.
+const mapExtractingPercentagesWithTooltipInfo = (accumulatedYields, key) => accumulatedYields.
 map(
```

Comment:
Lo normal es el punto en la nueva línea jaja, ese punto final se ve raro jeje
```suggestion
const mapExtractingPercentagesWithTooltipInfo = (accumulatedYields, key) => accumulatedYields
  .map(
```

Reply:
Me lo llevo